src/cli/index.js

```diff
@@ -62,6 +66,8 @@ export default (config, http) => {
     console.log(`export ALLOWED_EMAIL_DOMAINS=${conf.emailDomains}`);
     console.log(`export HARVEST_ACCESS_TOKEN=${conf.harvestAccessToken}`);
     console.log(`export HARVEST_ACCOUNT_ID=${conf.harvestAccountId}`);
+    console.log(`export HARVEST_ACCESS_TOKENS=${conf.harvestAccessTokens}`);
+    console.log(`export HARVEST_ACCOUNT_IDS=${conf.harvestAccountIds}`);
```

Comment:
Is there some specific format for these token and id lists? Like "company1:token1,company2:token2" and "company1:id1,company2:id2"?

Reply:
Yeah, that was the format. Transformed to object in settings.

Reply:
Documented format to readme